src/pages/Home.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Home() {
+  return (
+    <div className='home'>
+      <h1>Home</h1>
+    </div>
+  );
+}
+
```

Comment:
These pages are a good base, we should find a good template to fill out the location cards